### PR TITLE
New pylint exercice

### PR DESCRIPTION
**Pull request by carsofferrei**

### CODE REVIEW

File: assignments/week2.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Car:
+    def __init__(self, color):
+        self.color = color
+
+my_car = Car('blue')
+
+def crash(car1) :
+    car1.color = 'burnt'
+
+crash(Car('red'))
```
Comment (MrCordeiro) [inline assignments/week2.py:11]: The newline is missing

File: assignments/week2.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Car:
```
Comment (MrCordeiro) [inline assignments/week2.py:1]:
```suggestion
class Car:
    """Creates a car"""
```

**Review by MrCordeiro — CHANGES_REQUESTED** [review]

File: assignments/week2.py (hunk 1 above)
Comment (carsofferrei) [reply assignments/week2.py:11]: I will add this in the new version.

**Review by MrCordeiro — APPROVED** [review]
🌟 Approved!!

**Pull request merged**